### PR TITLE
Feat/miaw domain start10

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-3330
| License          | BSD 3-Clause

## Description

Update start10m order
 
## Dependencies
- [x] TRANS-25010

### CODE REVIEW

File: packages/manager/apps/web/client/app/domain/general-informations/webhosting-enable/domain-enable-web-hosting.controller.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            'domain_configuration_enable_web_hosting_success',
+          ),
+        ),
+      )
+      .catch((error) =>
+        this.Alerter.error(
+          this.$translate.instant(
+            'domain_configuration_enable_web_hosting_checkout_error',
+            {
+              message: error.message,
```
Comment (radireddy) [inline packages/manager/apps/web/client/app/domain/general-informations/webhosting-enable/domain-enable-web-hosting.controller.js:98]: may be message: get(error, 'data.message', error),

File: packages/manager/apps/web/client/app/domain/general-informations/webhosting-enable/domain-enable-web-hosting.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        </oui-step-form>
+        <oui-step-form
+            header="{{:: 'domain_configuration_enable_web_hosting_activation' | translate }}"
+            on-focus="$ctrl.loadCheckout()"
+            loading="$ctrl.isLoadingCheckout"
+            valid="$ctrl.contractsValidated"
+            submit-text="{{:: 'domain_configuration_enable_web_hosting_activate' | translate }}"
+            on-cancel="$ctrl.goBack()"
+        >
+            <ovh-contracts-summary
```
Comment (radireddy) [inline packages/manager/apps/web/client/app/domain/general-informations/webhosting-enable/domain-enable-web-hosting.html:90]: no payment method selection? how is the payment made?

Comment (marie-j) [reply packages/manager/apps/web/client/app/domain/general-informations/webhosting-enable/domain-enable-web-hosting.html:90]: Order is free so we autopay the purchase

**Comment by marie-j** [comment]
Updated to use `_.get` and add missing `ngInject`

File: packages/manager/apps/web/client/app/domain/general-informations/webhosting-enable/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  "domain_configuration_web_hosting_dns_info_MX": "Cette option vous permettra d'utiliser uniquement les adresses e-mail de votre hébergement sans modifier le pointage relatif à votre site web. Votre nom de domaine continuera de pointer sur votre site actuel, mais les adresses e-mail utiliseront désormais les serveurs d'OVH.",
+  "domain_configuration_enable_web_hosting_offer_price": "Tarification",
+  "domain_configuration_enable_web_hosting_offer_START_P1Y": "12 mois",
+  "domain_configuration_enable_web_hosting_activation": "Activation",
+  "domain_configuration_enable_web_hosting_activate": "Activer",
+  "domain_configuration_enable_web_hosting_success": "L'activation de l'hébergement a été réalisée",
+  "domain_configuration_enable_web_hosting_checkout_error": "Une erreur est survenue {{ message }}",
+  "domain_configuration_enable_web_hosting_error": "Une erreur est survenue lors de l'activation de votre hébergement. {{ message }}"
+
```
Comment (antleblanc) [inline packages/manager/apps/web/client/app/domain/general-informations/webhosting-enable/translations/Messages_fr_FR.json:20]: Extra empty line.

File: packages/manager/apps/web/client/app/domain/general-informations/webhosting-enable/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  "domain_configuration_enable_web_hosting_dns_configuration_A": "Entrée DNS A",
+  "domain_configuration_enable_web_hosting_dns_configuration_MX": "Entrée DNS MX",
+  "domain_configuration_web_hosting_dns_info_A": "Si vous disposez d'adresses e-mail sur votre nom de domaine mais d'aucun site internet, vous pouvez choisir de ne modifier que l'enregistrement A. De cette manière, votre nom de domaine pointera sur votre nouvel hébergement et vos adresses e-mails continueront de fonctionner sur votre service actuel.",
+  "domain_configuration_web_hosting_dns_info_MX": "Cette option vous permettra d'utiliser uniquement les adresses e-mail de votre hébergement sans modifier le pointage relatif à votre site web. Votre nom de domaine continuera de pointer sur votre site actuel, mais les adresses e-mail utiliseront désormais les serveurs d'OVH.",
```
Comment (antleblanc) [inline packages/manager/apps/web/client/app/domain/general-informations/webhosting-enable/translations/Messages_fr_FR.json:12]: We can take the opportunity here: 
```suggestion
  "domain_configuration_web_hosting_dns_info_MX": "Cette option vous permettra d'utiliser uniquement les adresses e-mail de votre hébergement sans modifier le pointage relatif à votre site web. Votre nom de domaine continuera de pointer sur votre site actuel, mais les adresses e-mail utiliseront désormais les serveurs d'OVHcloud.",
```

**Comment by marie-j** [comment]
I've only updated the translations to take care of the reviews

File: packages/manager/apps/web/client/app/domain/general-informations/webhosting-enable/domain-enable-web-hosting.html [hunk 5]
```diff
@@ -0,0 +1,98 @@
+<div ui-view>
+    <div>
```
Comment (JeremyDec) [inline packages/manager/apps/web/client/app/domain/general-informations/webhosting-enable/domain-enable-web-hosting.html:2]: useless `<div>` ?

**Comment by marie-j** [comment]
Removed useless `div`

**Pull request merged**